orangecontrib/text/widgets/icons/Twitter.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I prefer this 👍 Should we maybe make it also a bit smaller, since it is all black? Comparing to `Preprocess Text` and `Bag of Words` this icon seem a bit larger. @ajdapretnar what do you think?